packages/backend-common/src/reading/integration.test.ts

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ConfigReader } from '@backstage/config';
+import { getVoidLogger } from '../logging';
+import { UrlReaders } from './UrlReaders';
+
+const reader = UrlReaders.default({
+  logger: getVoidLogger(),
+  config: new ConfigReader({
+    // The tokens in this config provide read only access to the backstage-verification repos
+    integrations: {
+      github: [
+        {
+          host: 'github.com',
+          token: '0dad1a92474b8de2af3cc77b44745605aeeaaa98',
```

Comment:
Hey! I think GitHub has a feature that any plain-text token gets disabled automatically on merge/commit. Let's see 😅

Reply:
Yep, could totally be a thing 😅 , I'll add a lil precaution because tbh it should be no problem having this token in the public. It has no scopes at all, so should just help with ratelimiting. There's ofc a potential problem with local test runs using up rate limits.

General thinking is to ship this and then see if we need to adjust things

Reply:
Token was already disabled btw :grin:

Reply:
Haha, must have received an email :D